### PR TITLE
docs(aws): add AWS console UI steps 

### PR DESCRIPTION
**Pull request by aleon1220**
1-A proposal to have a section option 3 that describes the steps to be taken in AWS console manually. I am happy to add the screenshots that match the generic names 

2-A new Halyard section with description of the general steps that should be taken inside halyard

### CODE REVIEW

File: setup/install/providers/aws/aws-ec2.md [hunk 1]
```diff
@@ -79,8 +79,24 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
...
 
-## Configure Halyard to use AccessKeys (if configured)
+There are 2 options here
+1. Using AWS IAM AccessKey and Secret
+Option number 1 is useful for creation of user with AWS Access Key and secret. This is a common configuration. 
+2. Using AWS IAM Roles
+Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
+
+## Halyard Configurations
+After the AWS IAM user, roles, policies and trust relationship has been set up, the next step is to add the AWS configurations to spinnaker via Halyard CLI.
```
Comment (317brian) [inline setup/install/providers/aws/aws-ec2.md]: capital 'S' for Spinnaker

File: setup/install/providers/aws/aws-ec2.md [hunk 2]
```diff
@@ -79,8 +79,24 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
...
+
+The steps are as following:
+* Access the Halyard Pod
```
Comment (317brian) [inline setup/install/providers/aws/aws-ec2.md]: These steps should be an ordered list. Also, can we be explicit about each step, e.g. 

```
Apply the configurations to spinnaker
```

should become

```
4. Apply the configurations to Spinnaker: `hal deploy apply`. 
```

Period at the end of each list item too.

**Review by 317brian — CHANGES_REQUESTED** [review]

**Review by aleon1220 — COMMENTED** [review]
thank you. I agree with the changes

**Review by aleon1220 — COMMENTED** [review]
It looks good to me.

**Comment by 317brian** [comment]
Hi @aleon1220, there isn't a commit with my suggested updates.

**Review by 317brian — APPROVED** [review]

**Pull request merged**